dips/models.py

```diff
@@ -283,21 +270,17 @@ class DIP(AbstractEsModel):
         related_name="dips",
         verbose_name=_("collection"),
         on_delete=models.CASCADE,
+        blank=True,
+        null=True,
     )
     dc = models.OneToOneField(DublinCore, null=True, on_delete=models.SET_NULL)
-    # The TaskResult created by 'django_celery_results' are not added
-    # to the database when the import task is called. Therefore, a proper
-    # model relation can't be made at that point. This field is used to
-    # track the asynchronous result id from the task call, as that id
-    # will match the `task_id` unique field from the TaskResult.
-    import_task_id = models.CharField(max_length=255, null=True, unique=True)
-    # Because a related TaskResult may not exist for two reasons: not created
-    # yet or deleted (for example by a clean task), an extra field is needed
-    # to know if there is an import in progress for the DIP. This field is also
-    # used to track the import task status and it's set to `PENDING` when the
-    # task is called from the `new_dip` view and to `SUCCESS` or `FAILURE` when
-    # the task ends from within the task's `after_return` method.
     import_status = models.CharField(max_length=7, null=True)
+    import_error_trace = models.TextField(blank=True, null=True)
```

Comment:
Yes, these traces are shown to only some types of users in the Folder pages to provide some information about the error. But showing the error message instead of the entire trace should be enough.